web/pgadmin/dashboard/static/js/Dashboard.jsx

```diff
@@ -695,6 +695,7 @@ function Dashboard({
         ? treeNodeInfo.server.user.can_signal_backend
         : false;
 
+    let active_sessions = dashData.filter((data)=>data.state === 'active');
```

Comment:
shouldn't this filter also include database name comparison with maintenance_database?

Reply:
I have copied this logic from the old code. If we put a filter for maintenance_database, we will not be able to terminate/cancel the query running on the query tool of the maintenance database. Also, we can not terminate the connection made with the maintenance database (default connection), the backend will throw the error which is handled.

Reply:
as per the logic, it is trying to check if there are more than one connection for the connection being terminated. But what if there are multiple DB connections?

Reply:
If you check line 732, the condition is if we have more than one active connection then we should be able to terminate.

Reply:
What I want to ask is - if there are 2 active connections in the dashboard with both connected to different databases, will it allow terminating the maintenance one?